src/device-registry/routes/v1/devices.js

```diff
@@ -37,7 +37,7 @@ router.put(
       .bail()
       .trim()
       .toLowerCase()
-      .isIn(["kcca", "airqo", "urban_better", "us_embassy", "nasa", "unep"])
+      .isIn(["kcca", "airqo", "urban_better", "usembassy", "nasa", "unep"])
```

Comment:
Hi @123MwanjeMike , thanks for this recommendation. 

We have already made `BIG` and `MANY` corresponding changes in other systems like the `pipeline`, `auth-service` and etc. So I guess this is a bit late to go back now......

Reply:
To provide more context, this was informed by some recent access control implementations in the auth-service implementation which required this corresponding change.

All in all, we can just take some learning and move on.....because going backwards will frustrate everyone yet we already have a lot on our plates.

Thanks again @123MwanjeMike